var log = function (message) {
    console.log(message);
};
var throwError = function (error) {
    throw new Error(error);
};
var processData = function (data) {
    log(data);
};
var errorHandlingScenario = function (error) {
    throwError(error);
};
console.log(log("Hello TypeScript"));
console.log(throwError("Test error"));
console.log(processData("sample data"));
//console.log(errorHandlingScenario("An unexpected error occured!"));
